bigquery_etl/stripe/__init__.py

```diff
@@ -212,7 +205,11 @@ def schema(resource: Type[ListableAPIResource]):
     "--time-partitioning-type",
     default=bigquery.TimePartitioningType.DAY,
     type=click.Choice(
-        [bigquery.TimePartitioningType.DAY, bigquery.TimePartitioningType.MONTH]
+        [
+            bigquery.TimePartitioningType.DAY,
+            bigquery.TimePartitioningType.MONTH,
+            bigquery.TimePartitioningType.YEAR,
```

Comment:
With the addition of a year option, should the `--date` parameter be updated to also accept just the `%Y` format?